src/eko/harmonics/log_functions.py

```diff
@@ -88,3 +89,103 @@ def lm13m1(n, S1, S2, S3):
         - (2 * (6 * (2 * S3 - 2 * zeta3) + zeta3)) / n
         + (2 * (6 * (2 * S3 - 2 * zeta3) + zeta3)) / (1 + n)
     )
+
+
+@nb.njit(cache=True)
+def lm11(n, S1):
+    """Mellin transform of :math:`\\ln(1-x)`
+
+    Parameters
+    ----------
+    n : complex
+        Mellin moment
+    S1:  complex
+        Harmonic sum :math:`S_{1}(N)`
+
+    Returns
+    -------
+    complex
+
+    """
+    return -S1 / n
+
+
+@nb.njit(cache=True)
+def lm13(n, S1, S2, S3):
+    """Mellin transform of :math:`\\ln^3(1-x)`
+
+    Parameters
+    ----------
+    n : complex
+        Mellin moment
+    S1:  complex
+        Harmonic sum :math:`S_{1}(N)`
+    S2:  complex
+        Harmonic sum :math:`S_{2}(N)`
+    S3:  complex
+        Harmonic sum :math:`S_{3}(N)`
+
+    Returns
+    -------
+    complex
+
```

Comment:
```
:math:`\mathcal{M}[\ln^3(1-x)](N)`
```

Reply:
don't we have a shortcut for Mellin trafo?

Reply:
no, it's not that long in the end...

Reply:
the actual fix is in https://github.com/N3PDF/eko/pull/137/commits/1ae40bbb505d48ef0f1115ffd26389d1a12eb2a9

Reply:
Ahh I missed that one, thanks for fixing it ...